.gitignore

```diff
@@ -3,3 +3,4 @@ composer.phar
 composer.lock
 /src/views
 /src/config/config.dev.php
+.idea
```

Comment:
This should probably be in the [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).